### PR TITLE
update encryption client

### PR DESCRIPTION
**Pull request by odinuv**
https://github.com/keboola/azure-key-vault-php-client/releases/tag/1.0.0
https://keboola.atlassian.net/browse/PS-688

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ before_script:
 script:
 - php ./vendor/bin/phpcs --standard=psr2 --ignore=vendor -n .
 - php ./vendor/bin/phpunit --coverage-clover build/logs/clover.xml
-- docker run --rm phpstan sh -c "composer require phpstan/phpstan:^0.9 && /code/vendor/bin/phpstan analyse --level=max ./src"
+#- docker run --rm phpstan sh -c "composer require phpstan/phpstan:^0.9 && /code/vendor/bin/phpstan analyse --level=max ./src"
```
Comment (ondrejhlavacek) [inline .travis.yml:15]: Co se mu nelíbí?

Comment (odinuv) [reply .travis.yml:15]: Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - Conclusion: don't install phpstan/phpstan 0.9.3
    - Conclusion: don't install phpstan/phpstan 0.9.2
    - Conclusion: don't install phpstan/phpstan 0.9.1
    - phpstan/phpstan 0.9 conflicts with webmozart/assert[1.9.1].
    - phpstan/phpstan 0.9 conflicts with webmozart/assert[1.9.1].
    - phpstan/phpstan 0.9 conflicts with webmozart/assert[1.9.1].
    - Installation request for phpstan/phpstan ^0.9 -> satisfiable by phpstan/phpstan[0.9, 0.9.1, 0.9.2, 0.9.3].
    - Installation request for webmozart/assert (locked at 1.9.1) -> satisfiable by webmozart/assert[1.9.1].

https://travis-ci.org/github/keboola/object-encryptor/jobs/713589046

Comment (odinuv) [reply .travis.yml:15]: webmozart se tam instaluje jako zavislost neceho co je zavislost neceho co je zavislost neceho na starym phpunitu 5.7, driv to fungovalo, ale ted uz to nejde, nechci to debugovat vic protoze to jsou vsechno verze ktery jsou davno eol

**Review by ondrejhlavacek — APPROVED** [review]
lgtm

**Pull request merged**